sklearn/metrics/cluster/_supervised.py

```diff
@@ -729,6 +730,13 @@ def v_measure_score(labels_true, labels_pred, *, beta=1.0):
     return homogeneity_completeness_v_measure(labels_true, labels_pred, beta=beta)[2]
 
 
+@validate_params(
+    {
+        "labels_true": ["array-like", None],
+        "labels_pred": ["array-like", None],
+        "contingency": [np.ndarray, "sparse matrix", None],
```

Comment:
Since we have an inner call to `check_array`, we would support array-like and not only `ndarray`.

```suggestion
        "contingency": ["array-like", "sparse matrix", None],
```

Could you update the docstring with the following as well:

```
    labels_true : array-like of shape (n_samples,), dtype=integral
        ...

    labels_pred : array-like of shape (n_samples,), dtype=integral
        ...

    contingency : {array-like, sparse matrix} of shape \
            (n_classes_true, n_classes_pred), default=None
        ...
```

Reply:
Updated.